workers/unity/Assets/Gdk/Core/Components/WorldCommandsTranslation.cs

```diff
@@ -300,7 +300,7 @@ public void OnCreateEntityResponse(CreateEntityResponseOp op)
         public void OnDeleteEntityResponse(DeleteEntityResponseOp op)
         {
             Entity entity;
-            if (!TryGetEntityFromRequestId(op.RequestId.Id, "CreateEntityResponse", out entity))
+            if (!TryGetEntityFromRequestId(op.RequestId.Id, "DeleteEntityResponse", out entity))
```

Comment:
Can use `op.GetType().FullName` I think? Less brittle to changes in the future

Reply:
That will allocate on the heap, but agree that it would be nice to not have it hardcoded